Sources/AppcuesKit/Data/Analytics/AutoPropertyDecorator.swift

```diff
@@ -55,14 +56,15 @@ internal class AutoPropertyDecorator: AnalyticsDecorating {
             currentScreen = nil
             previousScreen = nil
         case .group:
-            // group updates do not have auto props, don't manipulate anything in the properties
-            return tracking
+            // group updates only have this single auto prop, so add that and return early
+            decorated.properties = (decorated.properties ?? [:]).merging([
+                "_lastSeenAt": Date()
+            ])
```

Comment:
double checking - if clearing the group id (setting to nil) does this create properties with a value for the nil group, when there otherwise would not have been a `group_update` in the payload? I suspect we should not send any properties in that case.

Reply:
Good catch, I added a case for `nil` group ids